src/service/admin/mod.rs

```diff
@@ -1257,6 +1269,65 @@ impl Service {
                     )
                 }
             },
+            AdminCommand::Acl(AclCommand::Add { mode, hostname }) => {
+                let host = match Host::parse(&hostname) {
+                    Ok(host) => host,
+                    Err(error) => {
+                        return Ok(RoomMessageEventContent::text_plain(format!(
+                            "failed to parse hostname with error {}",
+                            error
+                        )))
+                    }
+                };
+                if let Err(error) = services().acl.add_acl(host.clone(), mode) {
+                    error!(
+                        "encountered {} while trying to add acl with host {} and mode {:?}",
+                        error, host, mode
+                    );
+                    RoomMessageEventContent::text_plain("error, couldn't add acl")
+                } else {
+                    RoomMessageEventContent::text_plain("successfully added ACL")
```

Comment:
When deleting an entry that doesn't exist, it will not throw an error:

![image](https://github.com/girlbossceo/conduwuit/assets/112147643/6ff05f43-463d-46ff-95a3-519f6d0ca4f7)

Reply:
can you show logs?

Reply:
Nothing in logs:

```
[root@girlcock conduit-test]# journalctl -xefu conduit-test | grep -i GREPFORME
Journal file /var/log/journal/6aebcdb609374a1483a4d2f0f2db144e/user-1000@83da8209e7f14527b0b3f3e28a1f9a84-0000000000000000-0000000000000000.journal corrupted, ignoring file.
2023-12-24T23:16:17.779987Z  INFO http_request{path=/_matrix/client/v3/rooms/:room_id/send/:event_type/:txn_id}:build_and_append_pdu{pdu_builder=PduBuilder { event_type: "m.room.message", content: RawValue({"body":"@conduit:conduit-test.transfem.dev: acl remove GREPFORME","format":"org.matrix.custom.html","formatted_body":"<a href=\"https://matrix.to/#/@conduit:conduit-test.transfem.dev\">@conduit:conduit-test.transfem.dev</a>: acl remove GREPFORME","m.mentions":{"user_ids":["@conduit:conduit-test.transfem.dev"]},"msgtype":"m.text"}), unsigned: Some({"transaction_id": String("m1703459777670.1")}), state_key: None, redacts: None } sender="@test123:conduit-test.transfem.dev" room_id="!CYXyHk4Mw1erEdtAGo:conduit-test.transfem.dev"}: ruma_state_res::event_auth: auth_check beginning for $thiswillbefilledinlater (m.room.message)
2023-12-24T23:16:17.780027Z DEBUG http_request{path=/_matrix/client/v3/rooms/:room_id/send/:event_type/:txn_id}:build_and_append_pdu{pdu_builder=PduBuilder { event_type: "m.room.message", content: RawValue({"body":"@conduit:conduit-test.transfem.dev: acl remove GREPFORME","format":"org.matrix.custom.html","formatted_body":"<a href=\"https://matrix.to/#/@conduit:conduit-test.transfem.dev\">@conduit:conduit-test.transfem.dev</a>: acl remove GREPFORME","m.mentions":{"user_ids":["@conduit:conduit-test.transfem.dev"]},"msgtype":"m.text"}), unsigned: Some({"transaction_id": String("m1703459777670.1")}), state_key: None, redacts: None } sender="@test123:conduit-test.transfem.dev" room_id="!CYXyHk4Mw1erEdtAGo:conduit-test.transfem.dev"}: ruma_state_res::event_auth: $thiswillbefilledinlater ev_type 0 usr 100
2023-12-24T23:16:17.780039Z  INFO http_request{path=/_matrix/client/v3/rooms/:room_id/send/:event_type/:txn_id}:build_and_append_pdu{pdu_builder=PduBuilder { event_type: "m.room.message", content: RawValue({"body":"@conduit:conduit-test.transfem.dev: acl remove GREPFORME","format":"org.matrix.custom.html","formatted_body":"<a href=\"https://matrix.to/#/@conduit:conduit-test.transfem.dev\">@conduit:conduit-test.transfem.dev</a>: acl remove GREPFORME","m.mentions":{"user_ids":["@conduit:conduit-test.transfem.dev"]},"msgtype":"m.text"}), unsigned: Some({"transaction_id": String("m1703459777670.1")}), state_key: None, redacts: None } sender="@test123:conduit-test.transfem.dev" room_id="!CYXyHk4Mw1erEdtAGo:conduit-test.transfem.dev"}: ruma_state_res::event_auth: allowing event passed all checks
```

<img width="574" alt="image" src="https://github.com/girlbossceo/conduwuit/assets/112147643/5c5ab8e4-e1be-4a31-8b34-88ca1d6ea964">

Reply:
should be ok now